app/components/certificate/description.tsx

```diff
@@ -0,0 +1,44 @@
+import { Flex, Heading, Text, HStack, VStack } from '@chakra-ui/react';
+
+interface DescriptionProps {
+  request: Boolean;
+  validFrom: Date;
+  validTo: Date;
+}
+
+export default function DescriptionSection(props: DescriptionProps) {
+  const { request, validFrom, validTo } = props;
+  let description: String = '';
+
+  switch (request) {
+    case true:
+      description =
```

Comment:
Don't embed logic into components when you can pass props (i.e., pass `description` down if its value can change).